ios/AppLovinMAX.mm

```diff
@@ -584,7 +584,7 @@ - (instancetype)init
 }
 
 // NOTE: Even though iOS is ok with `id` generic types, Android is not - so we wrap it via JSON/ReadableMap
-RCT_EXPORT_METHOD(setBannerLocalExtraParameter:(NSString *)adUnitIdentifier :(NSDictionary<NSString *, id> *)parameterDict)
+RCT_EXPORT_METHOD(setBannerLocalExtraParameter:(NSString *)adUnitIdentifier parameter:(NSDictionary *)parameterDict)
```

Comment:
`parameter` signals that it is singular, but it should be plural. Also, what's the typing of the `NSDictionary`?

Reply:
Updated.  The signatures were auto-generated based on the the spec file.

Reply:
That's interesting, how to autogenerate?